anaconda.py

```diff
@@ -376,7 +376,7 @@ def setup_environment():
             # The module-level _() calls are ok here because the language may
             # be set from the live environment in this case, and anaconda's
             # language setup hasn't happened yet.
-            # pylint: disable=found-_-in-module-class
+            # pylint: disable=W9902
```

Comment:
Please add a fixme to set this back to the name because we want to get rid of these ID's which don't give you a hint about what is ignored.

Reply:
Done!